frontend/template-partials/views/partials/session-timeout-warning.html

```diff
@@ -8,15 +8,31 @@
 data-url-redirect="/session-timeout" class="modal-dialog dialog" role="dialog"
   aria-live="polite" aria-labelledby="dialog-title" aria-describedby="at-timer">
   <div class="modal-dialog__inner">
+    {{^showSaveAndExit}}
     <h2 id="dialog-title" class="govuk-heading-l">
       {{#dialogTitle}}{{#t}}pages.session-timeout-warning.dialog-title{{/t}}{{/dialogTitle}}{{^dialogTitle}}Your page will time out soon{{/dialogTitle}}
     </h2>
```

Comment:
Tabbing suggestion for the lines within both `{{showSaveAndExit}}` blocks:

```suggestion
    {{^showSaveAndExit}}
      <h2 id="dialog-title" class="govuk-heading-l">
        {{#dialogTitle}}{{#t}}pages.session-timeout-warning.dialog-title{{/t}}{{/dialogTitle}}{{^dialogTitle}}Your page will time out soon{{/dialogTitle}}
      </h2>
```

Reply:
Thanks for pointing this out, I have amended this now but I moved the mustache tags to the same level as the parent div instead of adding the extra tab to the html tag - adding the extra tab to the html adds extra whitespace while the mustache tags are not read as html so can be at the same level as the parent div.